server/src/routes/jokes.ts

```diff
@@ -35,7 +35,8 @@ const get_a_joke = async (
   const rando_offset = Math.floor(Math.random() * joke_count)
   const found_joke = await joke_model.findOne().skip(rando_offset).exec()
   if (!found_joke) res.send({error: true, message: 'no jokes, no content'})
-  res.send({joke: found_joke.joke})
+  //added a else statement.. was throwing an error in the server console if found joke was null.
+  else res.send({joke: found_joke.joke})
```

Comment:
this is b/c i didn't return the `res.send({error: true, message: 'no jokes, no content'})`, that's the correct fix here